flipperdevices/flipperzero-firmware
# [FL-3591] Ultralight fix unlock with reader

# What's new

Fixed "Unlock with password" and "Unlock with reader" functions

# Verification 
On **DEV** flipper
1. Add manually -> NTAG213
2. Apply such commands through proxmark:
   hf mfu wrbl -b 43 -d DEADBEAF //set PWD
   hf mfu wrbl -b 44 -d 00000000 //set PACK
   hf mfu wrbl -b 41 -d 04000004 //Lock card from page 4
3. Run emulation of this NTAG213

On NFC flipper:
See videos in task https://flipperzero.atlassian.net/browse/FL-3591

# Checklist (For Reviewer)
